charts/jaeger/templates/_helpers.tpl

```diff
@@ -572,6 +572,20 @@ Create pull secrets for all in one image
 {{- include "common.images.renderPullSecrets" (dict "images" (list .Values.allInOne.image) "context" $) -}}
 {{- end }}
 
+{{/*
+Create image name for v2 image
+*/}}
+{{- define "v2.image" -}}
```

Comment:
again, there shouldn't be any `v2` prefixes. There should be only major chart version bump.